nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/summary/ui/connection-status-listing/connection-status-table/connection-status-table.component.html

```diff
@@ -193,6 +193,16 @@
                                 </td>
                             </ng-container>
 
+                            <!-- Load Balance Status Column -->
+                            <ng-container matColumnDef="loadBalanceStatus">
+                                <th mat-header-cell *matHeaderCellDef mat-sort-header>
+                                    <div class="overflow-ellipsis overflow-hidden whitespace-nowrap">Load Balance</div>
```

Comment:
Recommend labeling this `Load Balancing`:
```suggestion
                                    <div class="overflow-ellipsis overflow-hidden whitespace-nowrap">Load Balancing</div>
```

Reply:
Oops - I was logged in to GitHub as a different user. The @nifiguy comments came from @markobean.